external-api/api-tests/v1/party-db.api-test.ts

```diff
@@ -39,12 +37,6 @@ describe('/party-db', () => {
 
       expect(status).toEqual(200);
     });
-
-    it('returns a 200 response with a valid companies house number', async () => {
```

Comment:
I removed it because I felt it wasn't necessary to test two different numeric company registration numbers. We already test the numeric and alphanumeric forms, which I felt covered our bases without unnecessary duplication.